src/status_im/contexts/keycard/create/events.cljs

```diff
@@ -20,7 +20,8 @@
   (rf/dispatch [:navigate-back])
   (rf/dispatch [:open-modal :screen/confirm-backup
                 {:masked-seed-phrase masked-seed-phrase
-                 :on-success         #(rf/dispatch [:keycard/create.phrase-backed-up %])}]))
+                 :on-success         #(rf/dispatch [:keycard/create.phrase-backed-up
```

Comment:
Hi, Please can you elaborate this change? (Doesn't seem related to PR)

Reply:
hey @Parveshdhull btw this is the fix for error you mention in your task, i'll move it to separate PR to merge asap